plugins/advisors/black-duck/src/main/kotlin/BlackDuck.kt

```diff
@@ -198,7 +198,7 @@ class BlackDuck(
 }
 
 internal fun VulnerabilityView.toOrtVulnerability(): Vulnerability {
-    val referenceUris = listOf(meta.href.uri(), *meta.links.map { it.href.uri() }.toTypedArray())
+    val referenceUris = setOf(meta.href.uri(), *meta.links.map { it.href.uri() }.toTypedArray())
```

Comment:
Commit message: What does "gracefully handle" actually mean? Why not just say "remove"?